ramin/array_method/array_method.js

```diff
@@ -0,0 +1,83 @@
+let array = []
+
+//---------------------------------//
+
+//push : add end
+array.push(1);
+array.push(2);
+array.push(3);
+
+//---------------------------------//
+//pop : delete from head
+array.pop();
+
+//---------------------------------//
+//shift : delete from head
+array.shift();
+
+//---------------------------------//
+//unshift : add head
+array.unshift(0);
+
+array.pop();
+array.pop();
+array.pop();
+
+array.push(1)
+array.push(2)
+array.push(3)
+
+//---------------------------------//
+//reverse
+array.reverse();
```

Comment:
please implement reverse function with toReversed()
What is the difference?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed

Reply:
okay sure.
but i think that not change the main array and return a changed array 
we should to declare a variable to receive.